<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkConnectomicsDataViewControls</class>
 <widget class="QWidget" name="QmitkConnectomicsDataViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="inputDataGroupBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lblWarning">
        <property name="styleSheet">
         <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
        </property>
        <property name="text">
         <string>Please select data!</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="inputImageOneLabel">
          <property name="text">
           <string>Image 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inputImageOneNameLabel">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="inputImageTwoLabel">
          <property name="text">
           <string>Image 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inputImageTwoNameLabel">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="networkCreationOptionsGroupBox">
     <property name="title">
      <string>Network Creation Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="mappingStrategyComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="networkifyPushButton">
     <property name="toolTip">
      <string>Create a network from a parcellation and a fiber image</string>
     </property>
     <property name="text">
      <string>Networkify</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createCorrelationMatrixPushButton">
     <property name="text">
      <string>Create Correlation Matrix</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="syntheticNetworkCreationPushButton">
     <property name="text">
      <string>Create Synthetic Networks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="syntheticNetworkOptionsGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Synthetic Network Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="syntheticNetworkComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="parameterOneLabel">
          <property name="text">
           <string>Parameter 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="parameterOneSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="parameterTwoLabel">
          <property name="text">
           <string>Parameter 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="parameterTwoDoubleSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>999.899999999999977</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parcellationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Anatomical labeling</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QmitkFreeSurferParcellationWidget" name="parcellationWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkFreeSurferParcellationWidget</class>
   <extends>QWidget</extends>
   <header>QmitkFreeSurferParcellationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
